naucse/templates/index.html

```diff
@@ -9,28 +9,28 @@
     <div class="card card-inverse space motivation">
       <img class="card-img" src="{{url_for('static', filename='pycon2016cz.png')}}" alt="Card image">
       <div class="card-img-overlay">
-        <p class="responsive-text card-text center">Nauč se Python, staň se superhrdinou nebo superhrdinkou a zachraň svět.<br>
-            Python je moderní programovací jazyk. Je univerzální – pohání weby i rakety.
-            Dobře se čte a dá se velice rychle naučit.</p>
+        <p class="responsive-text card-text center">Nauč se Python, staň se superhrdinou nebo superhrdinkou a&nbsp;zachraň svět.<br>
```

Comment:
@encukou That makes sense. This context is different.

About a and i, sure, I do that too with different words where it's not mandatory. I just wanted to point out it isn't (without asking what the reason was, shame on me).